.vscode/settings.json

```diff
@@ -3,5 +3,5 @@
     "yaml.schemas": {
         "./designs/design.json": ".design.yml"
     },
-    "python.linting.pylintEnabled": true
```

Comment:
can we keep this to false
i prefer using pylint from the command line it allows me to filter types of linting errors and more control
when VScode does it, it just highlights all my code and lowers my self-esteem which in turn lowers my productivity

Reply:
cool, it switched on automatically through my VSCode settings